components/Header.js

```diff
@@ -0,0 +1,24 @@
+// components/Header.js
+import Image from 'next/image'
+import MainHeaderLogo from '../assets/logo.svg'
+import ConnectWalletButton from "../components/ConnectWalletButton";
+import { Box } from '@chakra-ui/react'
+
+export default function Header() {
+  return (
+    <Box
+      m="40px"
+      mx="10%"
+      display="flex"
+      flexDirection="row"
+      alignItems="center"
+      justifyContent="space-between"
+    >
+      <Image
+        src={MainHeaderLogo}
+        alt="DEVELOPER DAO LOGO - SCHOOL OF CODE"
+      />
+      <ConnectWalletButton />
```

Comment:
I think we'd want to use a few props because we don't need the full account display

```suggestion
      <ConnectButton chainStatus="none" showBalance={false} />
```

Before:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/1954752/171528777-2076417f-003b-4374-a7da-c8eabf3e4b65.png">

After:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/1954752/171528788-4d7dede7-7b96-4fdd-957c-9bd13dff34ac.png">